### PR TITLE
added palindrome logic

### LINKED ISSUE #2: Add logic for palindrome in routes.js
Describe the bug
The logic for palindrome isn't currently present in the route '/palindrome'

To Reproduce

Route not working at the moment

Expected behavior
Should get a response whether the string or number is a palindrome or not

### PR DESCRIPTION
**Pull request by abhinav72610**
fix #2 
Added palindrome logic in routes.js used for loop along and arrow function

### CODE REVIEW

File: routes/routes.js [hunk 1]
```diff
@@ -6,6 +6,17 @@ router.post('/factorial', (req, res) => {
...
+ str = str.toLowerCase().replace(re, '');
+ var len = str.length;
+ for (var i = 0; i < len/2; i++) {
+   if (str[i] !== str[len - 1 - i]) {
+       return false;
+   }
+ }
+ return true;
+}
 })
```
Comment (ZiyanK) [inline routes/routes.js:20]: Here you have simply added a function directly. What is required to be done here is take take the str variable from the body of the request and apply the logic you have made, after which a response back as well.

**Review by ZiyanK — CHANGES_REQUESTED** [review]
Kindly make the required changes.

File: routes/routes.js (hunk 1 above)
Comment (abhinav72610) [reply routes/routes.js:20]: ok i will do it asap

**Comment by abhinav72610** [comment]
it had created merge conflicts it will be great if you resolve them and spare me if i have made any mistakes ..i'm complete begineer to open source

**Comment by ZiyanK** [comment]
We encourage you to learn these things since they will help you in the future as well. 
You can check out these links if you need some help on learning how to do so.
https://docs.github.com/en/free-pro-team@latest/github/using-git/resolving-merge-conflicts-after-a-git-rebase
https://docs.openstack.org/doc-contrib-guide/additional-git-workflow/rebase.html

Also make sure your local repository is up to date with this one before you make your final commit

**Pull request closed without merging**